packages/scandipwa/src/component/MyAccountOrderTableRow/MyAccountOrderTableRow.component.js

```diff
@@ -46,7 +46,7 @@ export class MyAccountOrderTableRow extends PureComponent {
                 <td>{ created_at }</td>
                 <td>{ status }</td>
                 <td block="hidden-mobile">
-                    { value ? formatPrice(value, currency) : '' }
+                    { typeof value === 'number' ? formatPrice(value, currency) : '' }
```

Comment:
Please show 0 for all falsy or empty values as well.

Reply:
If it is about ' ', then it is displayed only when loading, i.e. if I put some value there, the entire column will be filled with zeros when there is no other data yet

Reply:
But can we detect that loading state and not render anything during the loading?
Now it's an implicit feature and we will forget how it works after a month and we might break something.